app/src/main/java/nu/yona/app/api/manager/impl/AuthenticateManagerImpl.java

```diff
@@ -86,9 +86,9 @@ public boolean validateText(String string) {
      * @param mobileNumber user's mobile number
      * @return true if number is in expected format
      */
-    public boolean validateMobileNumber(String mobileNumber) {
+    public boolean isMobileNumberValid(String mobileNumber) {
 
-        return MobileNumberFormatter.validateMobileNumber(mobileNumber);
+        return MobileNumberFormatter.validate(mobileNumber);
```

Comment:
Sorry about this. The name should have been ``isValid``. I only focused on the last part of the name.